src/docs/product/sentry-basics/guides/discover/index.mdx

```diff
@@ -103,6 +103,12 @@ Each table cell offers a dynamic context menu that allows you to easily continue
 
 ![Context menu](013.png)
 
+### Errors by Hour or Day
+
+The `timestamp.to_hour` and `timestamp.to_day` fields round down the timestamp to the last UTC hour or day respectively. For example, an event with a `timestamp` value of Mar 12, 2021 2:26:50 PM UTC will have a `timestamp.to_hour` value of Mar 12, 2021 2:00:00 PM UTC and `timestamp.to_day` value of Mar 12, 2021 12:00:00 AM UTC. This allows you to roll up events. For example, you can show hourly hit counts for errors hit at least twice per hour:
```

Comment:
This shouldn't just always use UTC time. Like the `timestamp` field, this should respect the user's timezone preference.

Reply:
There's two parts here: calculation and formatting. For the calculation part, it's complicated to pass through the right time zone value, and raises questions about sharing queries. For the formatting part, that was broken on prod due to setting timezone to `Universal` instead of `UTC`, and was fixed in getsentry/sentry#24443.